core/src/test/java/org/frankframework/http/rest/ApiListenerServletTest.java

```diff
@@ -913,6 +926,33 @@ public void apiListenerWithRepeatableMessageAndGloballyDisabled() throws Excepti
 		assertTrue(result.containsHeader("pragma"));
 	}
 
+	@Test
+	public void apiListenerWithHeadMethodCall() throws Exception {
+		// Arrange
+		String uri = "/apiListenerWithHeadMethodCall";
+		Message repeatableMessage = Message.asMessage(new Message("{\"tralalalallala\":true}").asByteArray());
+		new ApiListenerBuilder(uri, List.of(Methods.HEAD))
+				.withResponseContent(repeatableMessage)
+				.build();
+
+		Map<String, String> headers = new HashMap<>();
+		headers.put("Accept", "application/json");
+		headers.put("content-type", "application/json");
+
+		// Act
+		Response result = service(createRequest(uri, Methods.HEAD, null, headers));
+
+		// Assert
+		assertEquals(200, result.getStatus());
```

Comment:
I do not see a check here that the content of the response is empty. That check is perhaps the most important check to validate if HEAD did what it is supposed to do.